pkg/controller/migplan/migplan_controller.go

```diff
@@ -362,8 +369,10 @@ func (r *ReconcileMigPlan) planSuspended(plan *migapi.MigPlan) error {
 		plan.Status.SetCondition(migapi.Condition{
 			Type:     Suspended,
 			Status:   True,
+			Reason:   SuspendedState,
```

Comment:
hrm, I am not sure if `Type: Suspended` and `Reason: SuspededState` is helpful or redundant.